### PR TITLE
Improve GPL Licensing

### PR DESCRIPTION
**Pull request by adghayes**
## Overview

Addresses #1439 for `vxsuite`, although I will need to do something similar for the other repo's.

Currently we have inconsistent licensing patterns:
- `LICENSE` files in some packages, not others
- References to GPL-v3 in some README's and `package.json`s but not all
- Some packages use GPL-v3 some use AGPL-v3

This PR attempts to standardize by:
- adding `LICENSE` file to the root and removing it everywhere else, as is common practice
- for Typescript packages, standardizing on a `license` field that is one of the SPDX official labels
- for Rust packages, setting the packages to inherit license from the `workspace` manifest
- standardizing on GPL-v3 rather than AGPL
- removing references to GPL from all README's except the root README, because I think that's a hard precendent to maintain

Let me know if you have any objections / suggestions to do this differently.

### CODE REVIEW

**Comment by adghayes** [comment]
Assigning @eventualbuddha as primary reviewer, @benadida for visibility.

**Review by eventualbuddha — APPROVED** [review]
I have no concerns about the mechanics of making this change. Assuming @benadida is okay with the implications of AGPL → GPL, I think this is fine.

The only suggestion I have is to update the `validate-monorepo` script to ensure that the fields are set as expected for all packages.

**Review by benadida — APPROVED** [review]
I am good with standardizing on GPL 3.0 only (which means not "or later").

**Comment by adghayes** [comment]
@eventualbuddha can you take a look at the monorepo validation I added?

Another inconsistency in our packages - we sometimes have an `author` specified as `VotingWorks Engineering <eng@voting.works>`, sometimes we don't. Do either of you have an inclination to standardize either way on that?

File: libs/@types/jsfeat/package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "name": "@types/jsfeat",
```
Comment (adghayes) [inline libs/@types/jsfeat/package.json:2]: @eventualbuddha does this package need to exist? I don't see it imported anywhere.

Comment (eventualbuddha) [reply libs/@types/jsfeat/package.json:2]: It does not. We stopped using `jsfeat` when we switched the ballot interpreter to Rust.

**Comment by eventualbuddha** [comment]
> Another inconsistency in our packages - we sometimes have an `author` specified as `VotingWorks Engineering <eng@voting.works>`, sometimes we don't. Do either of you have an inclination to standardize either way on that?

I don't really care. It's probably easier to leave it out, and as long as the tooling we use doesn't complain then let's remove it.

**Review by eventualbuddha — APPROVED** [review]
`validate-monorepo` changes LGTM 👍

**Pull request merged**